.github/workflows/update-deps.yml

```diff
@@ -47,6 +49,8 @@ jobs:
 
 
       - name: Update Nix flake inputs from submodules
+        env:
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I don't think this is needed, has not been needed anywhere else. Just for when the repo is being cloned.